src/scalar.jl

```diff
@@ -181,6 +181,7 @@ Base.:∘(t::ScalarTransform, ct::CompositeScalarTransform) = CompositeScalarTra
 Base.:∘(ct::CompositeScalarTransform, t::ScalarTransform) = CompositeScalarTransform((ct.transforms..., t))
 Base.:∘(ct1::CompositeScalarTransform, ct2::CompositeScalarTransform) = CompositeScalarTransform((ct1.transforms..., ct2.transforms...))
 Base.:∘(t::ScalarTransform, tt::Vararg{ScalarTransform}) = CompositeScalarTransform((t, tt...))
+const compose = Base.:∘
```

Comment:
I wonder if aliasing the whole of `∘` makes sense here; perhaps just alias and `CompositeScalarTransform`?

Reply:
Good call, it was trivial to replace this with a direct constructor.